### ISSUE TITLE
Move to producer role - terraform-provider-vault

### ISSUE THREAD

**Issue by zhenik** [fredrikhgrelland/vagrant-hashistack-template#58]
## Description
Reference log https://github.com/fredrikhgrelland/vagrant-hashistack/runs/1274213015?check_suite_focus=true#step:9:680

Change https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/master/template_example/example/standalone/providers.tf#L4

from 
```hcl
data "vault_generic_secret" "nomad_secret_id" {
  # Set count of this data source to 1 if ACLs are enabled in Nomad, and 0 if not
  count = var.nomad_acl ? 1 : 0
  path  = "nomad/creds/write"
}
```

to
```hcl
data "vault_generic_secret" "nomad_secret_id" {
  # Set count of this data source to 1 if ACLs are enabled in Nomad, and 0 if not
  count = var.nomad_acl ? 1 : 0
  path  = "nomad/creds/producer-role"
}
```

Related to [PR #414](https://github.com/fredrikhgrelland/vagrant-hashistack/pull/414)

**Comment by claesgill** [comment]
This got fixed in PR #75. However, shouldn't `path  = "nomad/creds/producer-role"` instead?

**Comment by zhenik** [comment]
> This got fixed in PR #75. However, shouldn't `path = "nomad/creds/producer-role"` instead?

I have updated the issue post

**Comment by claesgill** [comment]
It's correct already: https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/master/template_example/example/standalone/providers.tf#L4. Closing.
